File: dataset/human/c/file_2587.c
#include <stdio.h>

int main(void){
  char S[10] = {};
  int i, j, k;
  scanf("%s", S);
  if(S[0] != S[1] && S[1] != S[2] && S[2] != S[0]){
    printf("Yes\n");
  }else{
    printf("No\n");
  }
  return 0;
}

File: dataset/human/java/file_2587.java
import java.util.*;

class Main {
    public static void main(String[] args) {
        new Solver().run();
    }
}

class Solver {

    private String minimize(String s) {
        String buf = new String(s);
        char[] a = buf.toString().toCharArray();
        Arrays.sort(a);
        return new String(a);
    }

    private String maximize(String s) {
        String buf1 = new String(s);
        char[] a = buf1.toString().toCharArray();
        Arrays.sort(a);
        return new StringBuilder(new String(a)).reverse().toString();
    }

    public void run() {
        Scanner sc = new Scanner(System.in);
        String s = sc.next();
        String t = sc.next();

        String sd = minimize(s);
        String td = maximize(t);

        int ret = sd.compareTo(td);
        if (ret < 0) {
            System.out.println("Yes");
        } else {
            System.out.println("No");
        }

        return;
    }

}
